pkg/git/git.go

```diff
@@ -146,8 +170,28 @@ func GetGitPlan(plan string, localShuttleDirectoryPath string, uii *ui.UI, skipG
 	return planPath, nil
 }
 
-func RunGitPlanCommand(command string, plan string, uii *ui.UI) {
+func cacheIsValid(planPath string) (bool, error) {
+	duration := os.Getenv("SHUTTLE_CACHE_DURATION_MIN")
+	if duration == "" {
```

Comment:
I think we should have some docs of the feature. Do you have any plans on how we will make sure the variable is set?

Reply:
For now I've left it as opt in, as such the config isn't required. 

If we want to require it, we can just flip the valid = false, to return an error instead.

Reply:
My plan was to distribute this through zplug

Reply:
Will add docs though =D